content/legal-documents/sla.md

```diff
@@ -185,11 +185,22 @@ Payment is due once during a Service Period and the Customer will be charged for
 
 ## 16. Service Level Objectives
 
+Managed OpenShift clusters has this SLA:
+
 | Covered Service | Monthly Uptime Percentage |
 | --- | --- |
 | Zonal Clusters OpenShift API | 99.00% |
 | Regional Clusters OpenShift API | 99.50% |
 
+Managed services and addons running on the OpenShift clusters has this SLA:
+
+| SLA Type | Commitment |
+| --- | --- |
+| Uptime SLA | Best effort - no strict guarantee |
+| Backup SLA | Daily backups |
```

Comment:
maybe we should say where applicable? e.g. we do for Postgres but not for ArgoCD?